### PR TITLE
fix tnfw deflection angle at x == 1

### PR DESCRIPTION
**Pull request by dangilman**
there were some strange lines in the TNFW function that were redundant and resulted in the wrong deflection angle at x == 1

### CODE REVIEW

**Comment by dangilman** [comment]
added the changes you requested and changed SPEMD -> PEMD in the optimizer class

File: test/test_LensModel/test_Optimizer/test_multi_plane.py [hunk 1]
```diff
@@ -135,10 +135,10 @@ def test_params(self):
 
     def test_fixed_routines(self):
 
-        sie = FixedPowerLaw_Shear(['SPEMD','SHEAR'],self.kwargs_lens_simple,self.x_pos_simple,self.y_pos_simple,
+        sie = FixedPowerLaw_Shear(['PEMD','SHEAR'],self.kwargs_lens_simple,self.x_pos_simple,self.y_pos_simple,
```
Comment (sibirrer) [inline test/test_LensModel/test_Optimizer/test_multi_plane.py:138]: can you change this model to 'SIE' or 'SPEP'? The reason is that the PEMD (former SPEMD) requires fastell4py which is not part of the testing (returns zeros). Sorry that I pick this up now.

**Comment by dangilman** [comment]
I can't swap out lens models because the class expects certain kwargs_lens arguments, but I can just take out the test that uses fastell

**Comment by sibirrer** [comment]
thanks! Now, there is less test coverage in the optimizer :(

**Comment by dangilman** [comment]
Is there a way to include test functions for profiles that require fastell? Not sure how else to proceed 

- Daniel

**Comment by sibirrer** [comment]
> Is there a way to include test functions for profiles that require fastell? Not sure how else to proceed - Daniel

You can put in PEMD models in the testing and they will not raise, they just don't do what they are supposed to do (you will not predict 4 images etc). You don't need to test the specific lens model but rather whether the optimizer does what it supposed to do. A SPEP model e.g. is effectively almost the same (same parameters but ellipticity in the potential) and all the solver tests on my side are validated with this model instead.

**Comment by dangilman** [comment]
ok well as long as the PEMP doesn't raise an error the tests should pass and it should be ok right?

**Comment by sibirrer** [comment]
The tests are still lower than previously...

**Comment by dangilman** [comment]
how about now? I've been meaning to re-write the whole Optimizer class, but right now I don't have time to do that

**Comment by sibirrer** [comment]
thank you! Sorry for the hassle and the persistence.

**Pull request merged**